t/ui/14-dashboard-parents.t

```diff
@@ -146,5 +146,28 @@ subtest 'filtering subgroups' => sub {
     isnt(scalar @{$driver->find_elements('opensuse test', 'link_text')}, 0, "child group 'opensuse test' present'");
 };
 
+subtest 'View grouped by group' => sub {
+    my $parent_group_id = $schema->resultset('JobGroupParents')->search(undef, {name => 'Test parent'})->first->id;
+    $driver->get('/parent_group_overview/'. $parent_group_id);
+    wait_for_ajax_and_animations;
+
+    $driver->find_element_by_id('grouped_by_group_tab')->click();
+    wait_for_ajax_and_animations;
+
+    is(
+        $driver->find_element_by_id('grouped_by_group_tab')->get_attribute('class'),
+        'active parent_group_overview_grouping_active',
+        'grouped by group link isn\'t active'
```

Comment:
better avoid contractions and write 'grouped by group link not active'

Reply:
yes, you're right